deploy/kubernetes/elastic-agent-standalone-kubernetes.yml

```diff
@@ -23,10 +23,10 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: elastic-agent
-          image: docker.elastic.co/beats/elastic-agent:7.12.0-SNAPSHOT
+          image: docker.elastic.co/beats/elastic-agent:8.0.0
           args: [
             "-c", "/etc/agent.yml",
-            "-e", "-d", "composable.providers.kubernetes",
+            "-e", "-d", "*",
```

Comment:
Should we have the debug selector for all on by default? That seems like it would produce probably more than it should.

Maybe remove it with a comment on how to add it?

Reply:
👍🏼